criteria/version-control-and-history.md

```diff
@@ -12,7 +12,7 @@ order: 6
 * Every commit message MUST link to discussions and issues wherever possible.
 * You SHOULD group relevant changes in commits.
 * You SHOULD mark different versions of the codebase, for example using revision tags or textual labels.
-* You SHOULD prefer file formats that can easily be version controlled.
+* You SHOULD prefer file formats where the changes of the files can be easily viewed and understood in the version control system.
```

Comment:
2 options, with slightly different nuance. The first probably fulfills #316 slightly better, but is a clunky sentence.

```
* You SHOULD prefer file formats where the changes within the files can be easily viewed and understood in the version control system.
```
or:

```
* You SHOULD prefer file formats where changes can be easily viewed and understood in the version control system.
```

Reply:
Yeah, I see what you mean. For me, I think the "clunkiness" is probably worth it for the explicitness of fulfilling #316.